### ISSUE TITLE
tools/funcslower.py:support for the trace of nested functions

### ISSUE THREAD

**Issue by chenyuezhou** [iovisor/bcc#3448]
`tools/funcslower.py`

```
NOTE: This tool cannot trace nested functions in the same invocation
due to instrumentation specifics, only innermost calls will be visible.
```

Sometimes the slowness of the function is caused by too much nesting. Would it be better to calculate the elapsed time of the outermost layer

**Comment by yonghong-song** [comment]
Yes, you implemented the functionality for `funclatency.py`. Maybe you can give a try to `funcslower.py` as well?

**Comment by chenyuezhou** [comment]
Yes, I'll give it a try.

**Comment by rohansuri** [comment]
@yonghong-song does nested function mean for example, if I'm tracing `vfs_read`, it would automatically also print traces for `xfs_file_read_iter`, `submit_bio` if they cross the threshold?

**Comment by yonghong-song** [comment]
yes, if tracing `vfs_read`, the latency should include `xfs_file_read_iter`/`submit_bio` etc. But it won't print xfs_file_read_iter/submit_bio etc. If you want to trace these functions latency, you need specifically trace these functions.

**Comment by rohansuri** [comment]
thanks @yonghong-song. Is it easy to to write something that gets back the latencies for deeper stacks as well? Even if the function initially supplied was `vfs_read`, but a call deeper is also crossing the threshold, report that too?
